tests/test_templates_parameters.py

```diff
@@ -0,0 +1,197 @@
+
+    def test_pars_stronglyentangling_layers_dimensions(self, n_subsystems, n_layers):
+        """Confirm that the pennylane.templates.utils.parameters_stronglyentangling_layers()
+         returns an array with the right dimensions."""
+
+        a = (n_layers, n_subsystems, 3)
+        p = parameters_stronglyentangling_layers(n_layers=n_layers, n_wires=n_subsystems, seed=0)
+        dims = [p_.shape for p_ in p]
+        assert dims == [a]
+
+    def test_pars_stronglyentangling_layers_range(self, seed):
+        """Confirm that pennylane.templates.utils.parameters_stronglyentangling_layers()
+        samples from the right distributions."""
+
+        p = parameters_stronglyentangling_layers(n_layers=2, n_wires=1000, uniform_min=-2, uniform_max=1, seed=seed)
+        p_av = [np.mean(p_) for p_ in p]
+        p_std = [np.std(p_) for p_ in p]
+        assert np.isclose(p_av, [-0.5], atol=0.13)
+        assert np.isclose(p_std, [0.86], atol=0.13)
+
+
+class TestParsRandom:
+    """Tests the pennylane.templates.parameters methods for a random circuit."""
+
+    def test_pars_random_layer_dimensions(self, n_subsystems, n_rots):
+        """Confirm that the pennylane.templates.utils.parameters_random_layer()
+         returns an array with the right dimensions."""
+
+        if n_rots is None:
+            n_rots = n_subsystems
+
+        a = (n_rots, )
+        p = parameters_random_layer(n_wires=n_subsystems, n_rots=n_rots, seed=0)
+        dims = [p_.shape for p_ in p]
+        assert dims == [a]
+
+    def test_pars_stronglyentangling_layer_range(self, seed):
+        """Confirm that pennylane.templates.utils.parameters_random_layer()
+        samples from the right distributions."""
+
+        p = parameters_random_layer(n_wires=1000, uniform_min=-2, uniform_max=1, seed=seed)
+        p_av = [np.mean(p_) for p_ in p]
+        p_std = [np.std(p_) for p_ in p]
+        assert np.isclose(p_av, [-0.5], atol=0.13)
+        assert np.isclose(p_std, [0.86], atol=0.13)
+
+    def test_pars_random_layers_dimensions(self, n_subsystems, n_layers, n_rots):
+        """Confirm that the pennylane.templates.utils.parameters_random_layers()
+         returns an array with the right dimensions."""
+
+        if n_rots is None:
+            n_rots = n_subsystems
+
+        a = (n_layers, n_rots)
+        p = parameters_random_layers(n_layers=n_layers, n_wires=n_subsystems, n_rots=n_rots, seed=0)
+        dims = [p_.shape for p_ in p]
+        assert dims == [a]
+
+    def test_pars_stronglyentangling_layers_range(self, seed):
+        """Confirm that pennylane.templates.utils.parameters_random_layers()
+        samples from the right distributions."""
+
+        p = parameters_random_layers(n_layers=2, n_wires=1000, uniform_min=-2, uniform_max=1, seed=seed)
+        p_av = [np.mean(p_) for p_ in p]
+        p_std = [np.std(p_) for p_ in p]
+
+        assert np.isclose(p_av, [-0.5], atol=0.13)
+        assert np.isclose(p_std, [0.86], atol=0.13)
```

Comment:
```suggestion
        assert np.isclose(p_std, [0.86], atol=0.13)

```
This test looks great!

Edit: just noticed that `atol=0.13` seems a bit arbitrary/magic number-ish?

Reply:
Yes, it is a test of a mean value, and 0.13 is hand picked. You think that's not good? The seed is fixed and everything is deterministic!

Reply:
Oh, just stuck out as very arbitrary, I'm used to seeing tolerances of 1e-3, 0.01, 0.1, 0.05 etc. Does 0.1 not pass the test?

Reply:
Not with the current seed I think...

Reply:
Not with the current seed I think...